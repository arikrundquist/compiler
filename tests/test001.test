"this is a test"
"testing |\t|\r|\n|\"|\'|\\| escape sequences"
