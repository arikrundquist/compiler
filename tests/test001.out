STRING	OBJECT	this is a test
STRING	OBJECT	testing |	||
|"|'|\| escape sequences
